drgn/commands/_builtin/crash/_bpf.py

```diff
@@ -0,0 +1,271 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+# ebpf-related commands.
+
+import argparse
+from typing import Any, Generator, Optional, Tuple
+
+from drgn import Program
+from drgn.commands.crash import crash_command
+from drgn.helpers.common.type import enum_type_to_class
+from drgn.helpers.linux import (  # type: ignore[attr-defined]
+    bpf_map_for_each,
+    bpf_prog_for_each,
+    hlist_for_each_entry,
+)
```

Comment:
Import these from the specific submodules where they're defined (`drgn.helpers.linux.bpf` and `drgn.helpers.linux.list`) to avoid the need for the `type: ignore`.